pkg/volume/csi/csi_plugin.go

```diff
@@ -487,15 +487,10 @@ func (p *csiPlugin) ConstructVolumeSpec(volumeName, mountPath string) (*volume.S
 		switch {
 		case mode == ephemeralDriverMode:
```

Comment:
I don'd like `case` just with one branch, please simplify it even more. And add comments about falling through to persistent mode.